dbt_style_guide.md

```diff
@@ -103,19 +164,71 @@ select * from filtered_events
 ## SQL style guide
 
 - Use trailing commas
-- Indents should be four spaces (except for predicates, which should line up with the `where` keyword)
-- Lines of SQL should be no longer than 80 characters
+
+- Indents should be two spaces (except for predicates, which should line up with the `where` keyword)
+
+- Lines of SQL should be no longer than 80 characters and new lines should be used to ensure this.  
+  Example:
+  ```sql
+  sum(case
+    when order_status = 'complete'
+    then order_total

```

Comment:
personally I think 'then' should be indented. If you have multiple 'when' statements it makes it easier to read

Reply:
For maximum readability:

```sql
sum(
    case
        when order_status = 'complete'
            then order_total
    end
) as monthly_total,
```

Reply:
In that case, I would actually find it clearer to write:
```sql
sum(
    case when order_status = 'complete' then order_total end
) as monthly_total,
```

Reply:
Maybe better to have a longer case/when statement for this example

Reply:
Agree - this isn't a hard rule on where to indent, it's talking more about using new lines to make it fit. I do think there's another rule around here that does state it. I'll update this to be a longer case statement.

Reply:
Changed - [Commit](https://github.com/dbt-labs/corp/pull/63/commits/c48e5e39efd07756943eb8650dce63bfc5c57a3b)